src/test/java/org/apache/sling/distribution/queue/impl/resource/ResourceQueueIT.java

```diff
@@ -48,6 +49,7 @@
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
+@Ignore
```

Comment:
How does ignoring this test relate to SLING-8591 ?

Reply:
Also a leftover .. When I run the tests in eclipse this test kind of blocks .. so I have to ignore it ... and not forget to remove the ignore on commit. Actually if you have the chance maybe you could check if the test also blocks for you inside eclipse or do you use intellij?